### DISCUSSION TITLE
How to fix broken cache links for remote media after turning off "Cache remote files"?

### DISCUSSION THREAD

**Discussion by S0ulf3re** [misskey-dev/misskey#9062] [Q&A]
I had to make multiple adjustments to my object storage and had toggled "Cache remote files" a few times during the process. But now that I have everything setup. I still have media that keeps going to one of two places,

1. Misskey's files folder (which I deleted to free up space because I thought I wouldn't need it since Misskey could retrieve it directly from other instances)
2. An old object storage bucket which no longer exists because I deleted it

Safari's console is showing a bunch of 404 errors. Thatonecalculator said I could forcefully refresh it by clicking the user, the three dots -> about -> Update remote user information. But that doesn't seem to work. I am also technically running calckey as well, but that shouldn't be too much of a difference.

<img width="1440" alt="Screen Shot 2022-08-21 at 06 37 26" src="https://user-images.githubusercontent.com/50301900/185787101-58dedac7-4ab9-4987-b445-2863dc39dc07.png">

<img width="1366" alt="Screen Shot 2022-08-21 at 06 40 11" src="https://user-images.githubusercontent.com/50301900/185787163-2e2eb089-bb07-4db7-90d3-63f807d7f01a.png">

I'm wanting to turn off caching and go straight to grabbing remotely. How can I force misskey to find all of the missing profile pictures?

**Comment by geofflambeth** [comment]
I am encountering the same issue, also seemingly created while adjusting the object storage settings. I would love to find a way to clear these out or reset them—updating remote user information also does not work on my instance.

**Reply by geofflambeth** [reply]
Just noting that while I don’t have a large-scale solution, I believe I’ve found a workaround that (while inconvenient) doesn’t require as much overhead as the workaround identified in the original post:

1. Visit the “about” admin page for a given user where the issue has arisen
2. Navigate to “files” and proceed to delete all the images directing to old destinations (like object storage)
3. Then, use the “update remote user information” option discussed above.

I think this workaround is enough to solve my issue, but I only had a few users (maybe 10 minutes time?) with outdated object storage info. This would clearly not be a feasible solution for a larger issue.
